File: Solutions/58. Length of Last Word/58. Length of Last Word.go
func lengthOfLastWord(s string) int {
    s+=string(' ')
    tmp:=""
    for i:=0;i<len(s);i+=1{
        if len(tmp)>0 && string(s[i])!=string(' '){
            tmp=""
        }
        for string(s[i])!=string(' '){
            tmp+=string(s[i])
            i+=1
        }
    }
    if len(tmp)>0{
        return len(tmp)
    }
    return 0
}

File: Solutions/58. Length of Last Word/58. Length of Last Word.java
class Solution {
    public int lengthOfLastWord(String s) {
        s+=' ';
        String tmp="";
        for(int i=0;i<s.length();++i){
            if(tmp.length()>0 && s.charAt(i)!=' ')
                tmp="";
            while(s.charAt(i)!=' '){
                tmp+=s.charAt(i);
                ++i;
            }
        }
        return tmp.length()>0?tmp.length():0;
    }
}